src/oscontainer.py

```diff
@@ -70,7 +74,7 @@ def oscontainer_extract(containers_storage, src, dest,
 # with it.
 def oscontainer_build(containers_storage, src, ref, image_name_and_tag,
                       base_image, push=False, tls_verify=True, cert_dir="",
-                      inspect_out=None):
+                      authfile="", inspect_out=None):
```

Comment:
Generally new args (even kwargs) should come at the end, but I think this is fine since the only current caller is within this file and is already updated.